README.md

```diff
@@ -1,22 +1,36 @@
-# MiniModelarDB
-MiniModelarDB is a model-based time series management system that is primarily
-built as a re-design and re-implementation of
-[ModelarDB](https://github.com/ModelarData/ModelarDB). It is implemented in
-[Rust](https://www.rust-lang.org/) and uses
-[DataFusion](https://github.com/apache/arrow-datafusion) as the query engine,
-[Apache Arrow](https://github.com/apache/arrow-rs) as the in-memory format, and
-[Apache Parquet](https://github.com/apache/arrow-rs) as the on-disk format. The
-primary goals of the project are to experiment with low-level optimizations
-that are not easily possible on the JVM, design methods for efficiently
-representing irregular time series using a model-based approach while only
-requiring users to specify an error bound, develop a query optimizer that
-allows queries to automatically exploit the model-based representation, and
-provide a single easy to use relational query interface. MiniModelarDB is
-designed for Unix-like operating systems and is tested on Linux.
-
-MiniModelarDB intentionally does not gather usage data. So, all users are highly
+# ModelarDB
+ModelarDB is an efficient high-performance time series management system. It
+provides state-of-the-art lossless compression, lossy compression, and query
+performance by representing time series using multiple different types of models
+such as constant and linear functions. These compressed time series can be
+efficiently queried using a relational interface and SQL without any knowledge
+about the model-based representation. A query optimizer automatically rewrites
+the queries to exploit the model-based representation.
+
+ModelarDB is designed to be cross-platform and is tested on Microsoft Windows,
+macOS, and Ubuntu. It is implemented in [Rust](https://www.rust-lang.org/) and
+uses [Apache Arrow
+Flight](https://github.com/apache/arrow-rs/tree/master/arrow-flight) for
+communicating with clients, [Apache Arrow
+DataFusion](https://github.com/apache/arrow-datafusion) as its query engine,
+[Apache Arrow](https://github.com/apache/arrow-rs) as its in-memory data format,
+and [Apache Parquet](https://github.com/apache/arrow-rs/tree/master/parquet) as
+its on-disk data format.
+
+The first prototype of ModelarDB was implemented using [Apache
+Spark](https://www.h2database.com/html/main.html), [Apache
+Cassandra](https://cassandra.apache.org/_/index.html), and
+[H2](https://www.h2database.com/html/main.html) and was developed as part of a
+[research project](https://github.com/skejserjensen/ModelarDB) at Aalborg
+University and later as an [open-source
+project](https://github.com/ModelarData/ModelarDB). While this JVM-based
+prototype validated the benefits of using a model-based representation for time
+series, it has been superseded by the much most efficient Rust-based
```

Comment:
superceded by *this* much *more* efficient ...

Reply:
Fixed.